artichoke-backend/src/extn/core/enumerable/enumerable.rb

```diff
@@ -0,0 +1,450 @@
+##
+# Enumerable
+#
+module Enumerable
+
+  def drop(size)
+    size = size.__to_int
```

Comment:
Methods that begin with an underscore (`_`) are mruby internal methods and are not part of Ruby Core. Can we reimplement this logic with methods in Ruby Core? Maybe `to_int` or `to_i`?